.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
+- [ ] I linked the related issue(s) in the corresponding commit logs.
+- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/master/README#L92-L119).
```

Comment:
If you link to specific lines on the `master` branch then this link goes bad when the `README` is updated. If instead you had linked to the specific commit, this link would have stayed correct.